packages/flutter/lib/src/painting/strut_style.dart

```diff
@@ -44,6 +44,11 @@ import 'text_style.dart';
 /// leading plus ascent. Each line's spacing below the baseline will be at least as
 /// tall as the half leading plus descent.
 ///
+/// See also:
+///
+///  * [dart:ui.StrutStyle], the class in the [dart:ui] library that is
+///    encapsulated by this class and its subclasses.
```

Comment:
Is `encapsulated` correct in this case? Looks like the class is not really containing the dart:ui one, but rather a representation of it in widget land? @GaryQian